MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage.BindingContext>
+    <local:ViewModel x:Name ="viewModel"/>
+</ContentPage.BindingContext>
+
+<sfGrid:SfDataGrid x:Name="dataGrid"                   
+                   ItemsSource="{Binding OrdersInfo}">
+    <sfGrid:SfDataGrid.Columns>
+        <sfgrid:DataGridDateColumn Format="d"
+                                   HeaderText="Date"
+                                   MappingName="ShippedDate" />
+    </sfGrid:SfDataGrid.Columns>
+</sfGrid:SfDataGrid>
+{% endhighlight %}
+
+{% highlight c# %}
+dataGrid = new SfDataGrid();
+DataGridDateColumn dateColumn = new DataGridDateColumn()
+{
+    MappingName = "ShippedDate",
+    HeaderText = "Date",
+    Format = "d"
+};
+dataGrid.Columns.Add(dateColumn);
+{% endhighlight %}
+{% endtabs %}
+
+{% highlight c# %}
+// Model class
+public class Model
+{
+    private DateTime shippedDate;
+
+    public DateTime ShippedDate
+    {
+        get { return shippedDate; }
+        set
+        {
+            shippedDate = value;
+            RaisePropertyChanged("ShippedDate");
+        }
+    }
+}
+
+// ViewModel class
+public class ViewModel
+{
+    private List<DateTime> OrderedDates;
+
+    public ViewModel()
+    {
+        GetOrderDetails(50);
+    }
+
+    #region ItemsSource
+
+    private ObservableCollection<OrderInfo> ordersInfo;
+
+    public ObservableCollection<OrderInfo> OrdersInfo
+    {
+        get { return ordersInfo; }
+        set { this.ordersInfo = value; }
+    }
+
+    #endregion
+
+    #region ItemSource Generator
+
+     private List<DateTime> GetDateBetween(int startYear, int endYear, int count)
+     {
+        List<DateTime> date = new List<DateTime>();
+        Random d = new Random(1);
+        Random m = new Random(2);
+        Random y = new Random(startYear);
+        for (int i = 0; i < count; i++)
+        {
+            int year = y.Next(startYear, endYear);
+            int month = m.Next(3, 13);
+            int day = d.Next(1, 31);
+
+            date.Add(new DateTime(year, month, day));
+        }
+        return date;
+     }
+
+    public void GetOrderDetails(int count)
+    {
+        var orderDetails= new ObservableCollection<OrderInfo>();
+        this.OrderedDates = GetDateBetween(2000, 2014, count);
+        for (int i = 1; i <= count; i++)
+        {
+            var order = new OrderInfo()
+            {
+                ShippedDate = this.OrderedDates[i - 1],
+            };
+            orderDetails.Add(order);
+        }
+        ordersInfo = orderDetails;
+    }
+
+    #endregion
+}
+{% endhighlight %}
+
+![DateColumn](Images/column-types/date-column.PNG)
+
+## DataGridNumericColumn
+
+The `DataGridNumericColumn` inherits all the properties of `DataGridColumn`. It is used to display numeric data. To create `DataGridNumericColumn`, the property corresponding to the column in the underlying collection must be a numeric type(int, double, float, etc.). To create a `GridNumericColumn`, follow the code example:
```

Comment:
The `DataGridNumericColumn` inherits all the properties of the `DataGridColumn`. It is used to display numeric data. To create a `DataGridNumericColumn`, the property corresponding to the column in the underlying collection must be a numeric type (int, double, float, etc.). To create a `GridNumericColumn`, follow the code example: